.github/workflows/scripts/css_check.py

```
@@ -146,15 +269,31 @@ def check_embedded_styles(
+                # Check if we're in a Material-UI context that should be excluded
+                if is_in_mui_context(code_line, match.start()):
+                    continue
+                
                 if violation_type == "camelcase_css_property":
+                    # Check if it's a MUI component prop (e.g., fontSize="small")
+                    if is_mui_component_prop(code_line, match.start(), match.end()):
+                        continue
+                    
                     # Check if it's actually in a style context
+                    # Look for style-related keywords OR object literal context (braces)
                     preceding_text = code_line[: match.start()].strip()
```

Ruff (0.14.10)

Local variable `preceding_text` is assigned to but never used

Remove assignment to unused variable `preceding_text`

(F841)